cmd/spire-server/cli/x509/mint_test.go

```diff
@@ -108,12 +110,7 @@ func TestMintRun(t *testing.T) {
 	}))
 
 	server := new(fakeSVIDServer)
-	spiretest.StartGRPCSocketServer(t, util.DefaultSocketPath, func(s *grpc.Server) {
-		svidpb.RegisterSVIDServer(s, server)
-		bundlepb.RegisterBundleServer(s, server)
-	})
-
-	alternativeSocket := spiretest.StartGRPCSocketServerOnTempSocket(t, func(s *grpc.Server) {
+	spiretest.StartGRPCSocketServer(t, socketPath, func(s *grpc.Server) {
```

Comment:
As mentioned in the PR description, this test changed to always rely on a non-default socket location to prevent unit test failures if SPIRE server was running.